source/gui/settingsDialogs.py

```diff
@@ -2858,6 +2866,163 @@ def onBlinkCursorChange(self, evt):
 	def onNoMessageTimeoutChange(self, evt):
 		self.messageTimeoutEdit.Enable(not evt.IsChecked())
 
+
+class VisionSettingsPanel(SettingsPanel):
+	# Translators: This is the label for the vision panel
+	title = _("Vision")
+
+	def makeSettings(self, settingsSizer):
+		self.initialProviders = list(vision.handler.providers)
+		self.providerPanelInstances = []
+		settingsSizerHelper = guiHelper.BoxSizerHelper(self, sizer=settingsSizer)
+
+		# Translators: The label for a setting in vision settings to choose a vision enhancement provider.
+		providerLabelText = _("&Vision enhancement providers")
+
+		# Translators: Description in vision settings to choose a vision enhancement provider.
+		providersListDescriptionText = _(
+			"Providers are activated and deactivated as soon as you check or uncheck check boxes."
+		)
+		providersSizer = guiHelper.BoxSizerHelper(self, orientation=wx.VERTICAL)
+		providerListDescTextCtrl = providersSizer.addItem(wx.StaticText(self, label=providersListDescriptionText))
+		providerListDescTextCtrl.Wrap(self.scaleSize(544))

```

Comment:
@feerrenrut: What does 544 mean here?

Reply:
This is the width that is available to the text, we actually use the same value in the advanced settings panel. I would really prefer that we use `providerSizer.Size.Width`, except that during construction it is not correct, which I suspect has something to do with the order that we create things / add them to sizers etc. Unfortunately we couldn't use this value just anywhere, it is dependent on the nesting level of the text, and has to be checked manually.

Reply:
I decided to reword your comment here into an explanatory comment block in the text.